modules/admin-ui/src/test/java/org/opencastproject/adminui/endpoint/TestTasksEndpoint.java

```diff
@@ -126,56 +133,61 @@ public TestTasksEndpoint() throws Exception {
             anyObject(Map.class))).andReturn(wI2);
     replay(workspace, workflowService);
 
-    AssetManager assetManager = mkAssetManager(workspace);
+    org.opencastproject.assetmanager.api.AssetManager assetManager = mkAssetManager(workspace);
+
     MediaPackage mp1 = MediaPackageBuilderFactory.newInstance().newMediaPackageBuilder().createNew(new IdImpl("id1"));
     AttachmentImpl attachment = new AttachmentImpl();
     attachment.setFlavor(MediaPackageElements.PROCESSING_PROPERTIES);
     mp1.add(attachment);
 
     MediaPackage mp2 = MediaPackageBuilderFactory.newInstance().newMediaPackageBuilder().createNew(new IdImpl("id2"));
-    assetManager.takeSnapshot(AssetManager.DEFAULT_OWNER, mp1);
-    assetManager.takeSnapshot(AssetManager.DEFAULT_OWNER, mp2);
+
+    assetManager.takeSnapshot(org.opencastproject.assetmanager.api.AssetManager.DEFAULT_OWNER, mp1);
+    assetManager.takeSnapshot(org.opencastproject.assetmanager.api.AssetManager.DEFAULT_OWNER, mp2);
 
     this.setWorkflowService(workflowService);
     this.setAssetManager(assetManager);
     this.activate(null);
   }
 
-  AssetManager mkAssetManager(final Workspace workspace) throws Exception {
-    final PersistenceEnv penv = PersistenceEnvs.mk(mkEntityManagerFactory("org.opencastproject.assetmanager.impl"));
-    final Database db = new Database(null, penv);
-    return new AbstractAssetManager() {
-      @Override
-      public HttpAssetProvider getHttpAssetProvider() {
-        // identity provider
-        return new HttpAssetProvider() {
-          @Override
-          public Snapshot prepareForDelivery(Snapshot snapshot) {
-            return snapshot;
-          }
-        };
-      }
-
-      @Override
-      public Database getDb() {
-        return db;
-      }
-
+  org.opencastproject.assetmanager.api.AssetManager mkAssetManager(final Workspace workspace) throws Exception {
```

Comment:
Why not import this and use
```suggestion
  AssetManager mkAssetManager(final Workspace workspace) throws Exception {
```

Reply:
I think this was simply moved over from another spot, but yeah, importing will look a lot nicer. Will fix this.

Reply:
Done :)